src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/pages/Dashboard.vue

```diff
@@ -181,7 +181,7 @@
                   Canadian-certified ECE.
                 </p>
                 <div class="mt-auto">
-                  <router-link class="mt-4" to="test">Learn more</router-link>
+                  <router-link class="mt-4" to="icra/eligibility">Learn more</router-link>
```

Comment:
nitpik 

we can use the name of the route 

to="{name: "icra-eligibility"}"

helps us if we even need to change the URL to something else and I think we're using routes by name for most of the places in our application other than home.